unicoremc/models.py

```diff
@@ -436,9 +450,16 @@ def initiate_create_marathon_app(self, cmd):
             "cmd": cmd,
             "cpus": 0.1,
             "mem": 100.0,
-            "instances": 1
+            "instances": 1,
+            "labels": {
+                "domain": domain,
+                "app_type": self.app_type,
```

Comment:
Why are we adding the `app_type` twice? Once here and once in `domain.app_type` ?